packages/manager/modules/anthos/src/dashboard/general-information/general-information.html

```diff
@@ -160,43 +194,19 @@
                     data-description="{{:: $ctrl.tenant.storage }}"
                 >
                     <oui-tile-description>
-                        <p class="mb-1">
-                            <span data-ng-bind=":: $ctrl.netappStorage.totalSize | cucBytes:0:false:'MB' "></span>
-                        </p>
-                        <p class="mb-1">
-                            <span data-ng-bind=":: $ctrl.getStorageInfo()"></span>
-                        </p>
-                        <p class="mb-1">
-                            <oui-progress>
-                                <oui-progress-bar data-type="info"
-                                                  data-value="$ctrl.computeStoragePercent($ctrl.netappStorage.usedSize)">
-                                    <span></span>
-                                </oui-progress-bar>
-                                <oui-progress-bar data-type="success"
-                                                  data-value="$ctrl.computeStoragePercent($ctrl.netappStorage.reservedSize)">
-                                    <span></span>
-                                </oui-progress-bar>
-                            </oui-progress>
-
-                            <span>
-                               <oui-progress compact>
-                                  <oui-progress-bar type="info">
-                                    <span translate="anthos_tenant_dashboard_general_information_tile_resources_persistent_storage_type_system"></span>
-                                  </oui-progress-bar>
-                                </oui-progress>
-                            </span>
-                            <span>
-                               <oui-progress compact>
-                                  <oui-progress-bar type="success">
-                                    <span translate="anthos_tenant_dashboard_general_information_tile_resources_persistent_storage_type_svm"></span>
-                                  </oui-progress-bar>
-                                </oui-progress>
-                            </span>
-                        </p>
+                        <anthos-storage-usage
+                            data-storage-usage="$ctrl.storageUsage"
+                        ></anthos-storage-usage>
                     </oui-tile-description>
 
-                    <oui-action-menu aria-label="Actions" data-compact data-placement="end">
-                        <oui-action-menu-item data-on-click="$ctrl.onGoToOrderStorage()">
+                    <oui-action-menu
+                        aria-label="Actions"
```

Comment:
**nitpick:** Might be great if we can move this to a translation file.

Reply:
This is done